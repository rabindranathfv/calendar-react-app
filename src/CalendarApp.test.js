import { shallow } from "enzyme/build";
import { CalendarApp } from "./CalendarApp";

describe('CalendarApp::::', () => {
  it('render CalendarApp main component component properly', () => {
    let wrapper = shallow(<CalendarApp />);
    expect(wrapper).toMatchSnapshot();
  })
})