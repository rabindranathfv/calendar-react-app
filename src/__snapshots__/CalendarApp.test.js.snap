// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalendarApp:::: render CalendarApp main component component properly 1`] = `
<div>
  CalendarApp
</div>
`;
